xcp_d/workflows/base.py

```diff
@@ -361,26 +361,28 @@ def init_single_subject_wf(subject_id: str):
                 ])  # fmt:skip
 
     # Estimate head radius, if necessary
-    # Need to warp the standard-space brain mask to the anatomical space to estimate head radius
-    warp_brainmask = ApplyTransforms(
-        input_image=subj_data['anat_brainmask'],
-        transforms=[subj_data['template_to_anat_xfm']],
-        reference_image=subj_data[anat_mod],
-        num_threads=2,
-        interpolation='GenericLabel',
-        input_image_type=3,
-        dimension=3,
-    )
-    os.makedirs(config.execution.work_dir / workflow.fullname, exist_ok=True)
-    warp_brainmask_results = warp_brainmask.run(
-        cwd=(config.execution.work_dir / workflow.fullname),
-    )
-    anat_brainmask_in_anat_space = warp_brainmask_results.outputs.output_image
+    head_radius = config.workflow.head_radius
+    if config.workflow.head_radius == 'auto':
+        # Warp the standard-space brain mask to the anatomical space to estimate head radius
+        warp_brainmask = ApplyTransforms(
+            input_image=subj_data['anat_brainmask'],
+            transforms=[subj_data['template_to_anat_xfm']],
+            reference_image=subj_data[anat_mod],
+            num_threads=2,
+            interpolation='GenericLabel',
+            input_image_type=3,
+            dimension=3,
+        )
+        os.makedirs(config.execution.work_dir / workflow.fullname, exist_ok=True)
```

Comment:
Why does this directory need to be explicitly made?

Reply:
ohhh I see, because this is being run during the workflow build, not as part of the workflow itself